cmd/kube-scheduler/app/options/insecure_serving.go

```diff
@@ -108,7 +108,7 @@ func (o *CombinedInsecureServingOptions) ApplyToFromLoadedConfig(c *schedulerapp
 	return o.applyTo(c, componentConfig)
 }
 
-func updateAddressFromInsecureServingOptions(addr *string, is *controlleroptions.InsecureServingOptions) error {
+func updateAddressFromInsecureServingOptions(addr *string, is *apiserveroptions.DeprecatedInsecureServingOptions) error {
```

Comment:
Should we rename `updateAddressFromInsecureServingOptions` to `DeprecatedInsecureServingOptions`?

Reply:
done